scripts/tasks/task-annotate.py

```diff
@@ -0,0 +1,165 @@
+    parser.add_argument(
+        "-p", "--path", help="the directory to process"
+    )
+    parser.add_argument(
+        "-c", "--comment", help="A comment added to the annotation to help with search/removal", default="DBT:ANNOTATE"
+    )
+    parser.add_argument(
+        "-m", "--max", help="Maximum number of files to annotate"
+    )
+    parser.add_argument(
+        "-t", "--template", help="A template for the log statement", default='D_PRINTLN("{function}");'
+    )
+
+    return parser
+
+def remove_specific_lines(file_path, text_to_remove):
+    with open(file_path, 'r') as file:
+        lines = file.readlines()
+        filtered_lines = [line for line in lines if text_to_remove not in line]
+        if len(filtered_lines) < len(lines):
+            with open(file_path, 'w') as file:
+                file.writelines(filtered_lines)
+            print(f"Removed {len(lines) - len(filtered_lines)} lines from {file_path}")
+            return
+
+def remove_annotations(args):
+    for root, dirs, files in os.walk(args.path):
+        for file in files:
+            if file.endswith('.cpp'):
+                remove_specific_lines(os.path.join(root, file), args.comment)
+
+def is_function_definition(node):
+    """
+    Check if the node is a function definition.
+    """
+    if node.kind != clang.cindex.CursorKind.CXX_METHOD:
+        return False
+    for child in node.get_children():
+        if child.kind == clang.cindex.CursorKind.COMPOUND_STMT:
+            return True
+    return False
+
+
+
+def ensure_tags_exist():
+    result = subprocess.run(["ctags", "--version"])
+    ctags_binary = "ctags"
+    if result.returncode != 0:
+        def iter_ctags_binaries():
+            which_ctags_result = subprocess.run(["which", "-a", "ctags"], capture_output=True)
+            for line in which_ctags_result.stdout.decode('utf-8').splitlines():
+                yield line
+        print("Default ctags is erroring out on `ctags --version`. GNU standard ctags required. (brew install universal-ctags on MacOSX).")
+        ctags_binary = iterfzf(iter_ctags_binaries(), prompt = 'Please choose the appropriate ctags path: ')
+
+    ctags_invocation = [ctags_binary, "-R", "-x", "--c++-types=f", "--extras=q", "--format=1", 
+                    "src/deluge/gui/views" ,
+                    "src/deluge/deluge.cpp",
+                    "src/deluge/model/action",
+                    "src/deluge/model/clip", 
+                    "src/deluge/model/consequence",
+                    "src/deluge/model/drum",
+                    "src/deluge/model/instrument"]
+    print(f"invoking {' '.join(ctags_invocation)}")
+    result = subprocess.run(ctags_invocation, capture_output=True)
+    if result.returncode == 0:
+        with open(tags_file, "w+") as tags_file_pointer:
+            tags_file_pointer.write(result.stdout.decode('utf-8'))
+    else:
+        print(f"Failed to invoke {" ".join(ctags_invocation)}")
```

Comment:
```suggestion
        print(f"Failed to invoke {' '.join(ctags_invocation)}")
```
Looks like the python format check picked up an actual bug in this format string.